cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -777,12 +777,13 @@ makeTransactionBodyAutoBalance
   -> ProtocolParameters
   -> Set PoolId       -- ^ The set of registered stake pools
   -> UTxO era         -- ^ Just the transaction inputs, not the entire 'UTxO'.
+  -> ScriptValidity
```

Comment:
We should put a comment here similar to: "Mark script as expected to pass or fail validation"

Reply:
Done